pkg/scalers/datadog_scaler.go

```diff
@@ -272,29 +290,35 @@ func (s *datadogScaler) getQueryResult(ctx context.Context) (float64, error) {
 
 	series := resp.GetSeries()
 
-	if len(series) > 1 {
-		return 0, fmt.Errorf("query returned more than 1 series; modify the query to return only 1 series")
-	}
```

Comment:
I would log a warning in any case, as this is not Datadog official way of doing things. This enters unknown territory

Reply:
I would disagree with it being non-official - this method came directly from Datadog support as the best way to perform multiple queries (as it avoids rate limiting, which was the discussion in question with them).

I can log a warn though; which method is that? 😅 I'm new to go

Reply:
I think I've mitigated this by only logging an error if multi-query is used, and aggregator isn't set.
I've also removed mention of ```queryAggregator``` from the documentation examples that aren't specifically talking about multi-query - this should avoid accidental configuration of something people don't understand :)